articles/azure-monitor/containers/container-insights-enable-arc-enabled-clusters.md

```diff
@@ -114,10 +114,29 @@ az k8s-extension create --name azuremonitor-containers --cluster-name <cluster-n
 
```

Comment:
Add note in line 110 before the azurecli stating that 'Managed identity authentication will be default in k8s-extension version 1.43.0 or higher.'

Reply:
Added.